core/components/header/_actions/get-search-results.ts

```diff
@@ -1,67 +0,0 @@
-'use server';
```

Comment:
Now that I think about, we might want to keep this one since it'll be used for search. 🤔